userProfile/src/components/Form/form.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { Button, Gapped, Input, Select } from '@skbkontur/react-ui';
+import { UserFormData } from '../../index';
+
+type FormProps = { onSubmit: Function };
+type FormState = UserFormData;
+
+export class Form extends React.Component<FormProps, FormState> {
+    private cities = ['Екатеринбург', 'Качканар', 'Москва'];
+
+    render() {
+        return (
+            <form>
+                <h1>Информация о пользователе</h1>
+                <Gapped gap={20} vertical={true}>
```

Comment:
Атрибут `vertical` является булевым, а значит просто можно написать так
```suggestion
                <Gapped gap={20} vertical>
```